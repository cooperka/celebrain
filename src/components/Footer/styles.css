.Footer {
  width: 100%;
  height: 80px;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  bottom: 0;
  position:absolute;
}

.Footer div {
  margin: 0.4em;
  font-size: 1.2em;
}
