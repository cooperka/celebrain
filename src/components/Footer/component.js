// @flow

import * as R from 'ramda';
import classNames from 'classnames';
import React, { Component } from 'react';
import { withStyles } from 'material-ui';

import { size } from '../../constants';

import './styles.css';

type Props = {
  classes: Object,
};

class Footer extends Component<Props> {

  render() {
    const { classes } = this.props;

    return (
      <div className={classNames(classes.footer, 'Footer')}>
        <div>
          &copy; 2020 Kevin Cooper &bull;
          {' '}
          <a href="https://github.com/cooperka/neuro-name">View source</a>
        </div>
        <div>Images are sourced from Wikipedia and may be subject to copyright.</div>
      </div>
    );
  }

}

const styles = () => ({
  footer: {
    height: size.FOOTER_HEIGHT,
  },
});

export default R.compose(
  withStyles(styles),
)(Footer);
