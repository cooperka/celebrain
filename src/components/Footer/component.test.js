// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './component';

jest.mock('../SocialLink/component', () => 'SocialLink');

it('renders', () => {
  const tree = renderer.create(<Footer />);
  expect(tree.toJSON()).toMatchSnapshot();
});
