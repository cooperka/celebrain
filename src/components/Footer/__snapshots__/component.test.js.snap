// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="Footer-footer-1 Footer"
>
  <div>
    © 2020 Kevin Cooper •
     
    <a
      href="https://github.com/cooperka/neuro-name"
    >
      View source
    </a>
  </div>
  <div>
    Images are sourced from Wikipedia and may be subject to copyright.
  </div>
</div>
`;
